markdown/plext/using_pljava.html.md.erb

```diff
@@ -2,205 +2,222 @@
 
 	``` shell
-	$ hawq scp -f hawq_hosts myclasses.jar =:$GPHOME/lib/postgresql/java/
+	gpadmin@master$ hawq scp -f hawq_hosts myclasses.jar =:/usr/local/hawq/lib/postgresql/java/
 	```
-	The `hawq_hosts` file contains a list of the HAWQ hosts.
 
-3. Add the JAR files to the `pljava_classpath` configuration parameter. Refer to [Setting PL/Java Configuration Parameters](#setting_serverconfigparams) for the specific procedure.
+3. Add all JAR files to the `pljava_classpath` configuration parameter. Refer to [Setting PL/Java Configuration Parameters](#setting_serverconfigparams) for the specific procedure.
 
-5. (Optional) Your HAWQ installation includes an `examples.sql` file.  This script contains sample PL/Java functions that you can use for testing. Run the commands in this file to create and run test functions that use the Java classes in `examples.jar`:
+5. (Optional) Your HAWQ installation includes an `examples.sql` file. This script includes sample PL/Java functions that you can use to test PL/Java installation. Run the commands in this file to create and execute PL/Java user-defined functions that use the Java classes implemented in `examples.jar`. The database on which you run the script must have PL/Java enabled:
 
 	``` shell
-	$ psql -f $GPHOME/share/postgresql/pljava/examples.sql
+	gpadmin@master$ psql -d <dbname> -f $GPHOME/share/postgresql/pljava/examples.sql
 	```
 
 #### Configuring PL/Java VM Options
 
-PL/Java JVM options can be configured via the `pljava_vmoptions` server configuration parameter. For example, `pljava_vmoptions=-Xmx512M` sets the maximum heap size of the JVM. The default `-Xmx` value is `64M`.
+Configure PL/Java JVM options via the `pljava_vmoptions` server configuration parameter. For example, `pljava_vmoptions=-Xmx512M` sets the maximum heap size of the JVM. (The default `-Xmx` value is `64M`.)
 
 Refer to [Setting PL/Java Configuration Parameters](#setting_serverconfigparams) for the specific procedure to set PL/Java server configuration parameters.
-
 	
 ### <a id="uninstallpljava"></a>Disable PL/Java 
 
 To disable PL/Java, you should:
 
 1. Remove PL/Java support from each database in which it was added.
-2. Uninstall the Java JAR files.
+2. Uninstall/remove the Java JAR files.
 
 #### <a id="uninstallpljavasupport"></a>Remove PL/Java Support from Databases 
 
-For a database that no longer requires the PL/Java language, remove support for PL/Java by running the `uninstall.sql` script as the `gpadmin` user. For example, the following command disables the PL/Java language in the specified database:
+For a database that no longer requires the PL/Java language, remove support for PL/Java by running the `uninstall.sql` script as the `gpadmin` user. For example, the following command disables the PL/Java languages in the specified database:
```

Comment:
Edit:  Seems ok to remove "For a database that no longer requires the PL/Java language, " here.